### ISSUE TITLE
feature: EventTarget

### ISSUE THREAD

**Issue by make-github-pseudonymous-again** [zloirock/core-js#1039]
`EventTarget` does not exist in Node < 16. [Example](https://github.com/testing-library/user-event/issues/817).

Some existing polyfills:
  - https://github.com/benlesh/event-target-polyfill#readme
  - https://github.com/mattkrick/event-target-polyfill

**Comment by zloirock** [comment]
I'll think makes it sense or not, thanks.

**Comment by make-github-pseudonymous-again** [comment]
It's listed as experimental starting in Node 14.5 but I cannot find the flag to enable it (in Node 14.18).

**Comment by joshxyzhimself** [comment]
Hi, a bit similar, I was looking for a polyfill of Headers which has long been supported in browsers but just recently in v18 of Node.js. Its API is quite similar to URLSearchParams though.

- https://developer.mozilla.org/en-US/docs/Web/API/Headers
- https://nodejs.org/api/globals.html#class-headers

**Comment by ologbonowiwi** [comment]
We still need this?

I'm thinking about do some research on this one

**Comment by zloirock** [comment]
Yes, it's still required and can be useful for some other features.

**Comment by ologbonowiwi** [comment]
Should I build this on the top of which node version @zloirock?

**Comment by zloirock** [comment]
All possible.

**Comment by ologbonowiwi** [comment]
EventTarget API Specification: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget
Implementation example found on stack overflow: https://stackoverflow.com/a/51982980

**Comment by zloirock** [comment]
> ~~EventTarget API Specification: [developer.mozilla.org/en-US/docs/Web/API/EventTarget](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget)~~

EventTarget API Specification: https://dom.spec.whatwg.org/#interface-eventtarget

> Implementation example found on stack overflow: [stackoverflow.com/a/51982980](https://stackoverflow.com/a/51982980)

It's not the best place to look for polyfills code -)

**Comment by cristian-atehortua** [comment]
A nice implementation: https://github.com/mysticatea/event-target-shim

**Comment by johnzhou721** [comment]
Note that Events aren't even polyfilled: #354 — so we'd have to wait on that.  Once Events are implemented, this is going to be ismple.
